frontend/src/components/new-sightings-page/NewSightings.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { useEffect, useState } from "react";
```

Comment:
Not using either of these hooks
```suggestion
import React from "react";
```